### ISSUE TITLE
Calculating similarity as a Percentage

### ISSUE THREAD

**Issue by PradumanS** [deepinsight/insightface#963]
Is there any way to get percentage from the similarity value of face match in the test.py of deploy

**Comment by phuonglk** [comment]
Cosine similarity * 100%

**Comment by PradumanS** [comment]
Do you mean similarity is the confidence of face match ?

**Comment by corbanvilla** [comment]
Yes, because similarity = cosine(theta).... a perfect match would have an angle of 0 between the matrices. Cos(0) = 1.
